Python_PostBuild/Program.cs

```diff
@@ -14,29 +14,29 @@ static void Main(string[] args)
             if (args.Contains("--force"))
                 force = true;
 
-            // Install python
+            // Install python	
             Console.WriteLine("Installing python 3.7 embedded...");
             Compute.Install(force).Wait();
 
-            // Check the installation was successful 
+            // Check the installation was successful 	
             if (!Query.IsInstalled())
                 throw new SystemException("Could not install Python");
 
-            // Install pip
+            // Install pip	
             Console.WriteLine("Installing pip...");
             Compute.InstallPip();
 
-            // Check the pip installation was successful 
+            // Check the pip installation was successful 	
             if (!Query.IsPipInstalled())
                 throw new SystemException("Could not install pip");
 
-            // install project jupyter
+            // install project jupyter	
             Console.WriteLine("Installing jupyter...");
             Compute.PipInstall("jupyter");
             Compute.PipInstall("jupyterlab");
             Compute.PipInstall("pythonnet");
 
-            // install pyBHoM
+            // install pyBHoM	
```

Comment:
It feels so good when you haven't CTRL + K,  D and someone does it :1st_place_medal:

Reply:
I didn't mean to! 😸

Reply:
ahahahahhah I can see it might sound sarcastic, but I actually meant it :laughing: